tests/foreman/cli/test_satellitesync.py

```diff
@@ -230,11 +233,86 @@ def test_positive_export_rh_product(self):
         self.assertGreaterEqual(len(result.stdout), 1)
 
 
-class InterSatelliteSyncTestCase(CLITestCase):
-    """Implements InterSatellite Sync tests in CLI"""
+class ContentViewSync(CLITestCase):
+    """Implements Content View Export Import tests in CLI"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Create Directory for all CV Sync Tests in /tmp"""
+        super(ContentViewSync, cls).setUpClass()
+        if ssh.command('[ -d /tmp/CvSync ]').return_code == 0:
+            ssh.command('rm -rf /tmp/CvSync')
+        ssh.command('mkdir /tmp/CvSync')
+        cls.exporting_org = make_org()
+        cls.exporting_prod = gen_string('alpha')
+        product = make_product({
+            'organization-id': cls.exporting_org['id'],
+            'name': cls.exporting_prod
+        })
+        cls.exporting_repo = gen_string('alpha')
+        repo = make_repository({
+            'name': cls.exporting_repo,
+            'download-policy': 'immediate',
+            'product-id': product['id']
+        })
+        Repository.synchronize({'id': repo['id']})
+        cls.exporting_cv = gen_string('alpha')
+        content_view = make_content_view({
+            'name': cls.exporting_cv,
+            'organization-id': cls.exporting_org['id']
+        })
+        ContentView.add_repository({
+            'id': content_view['id'],
+            'organization-id': cls.exporting_org['id'],
+            'repository-id': repo['id']
+        })
+        ContentView.publish({u'id': content_view['id']})
+        content_view = ContentView.info({u'id': content_view['id']})
+        cls.exporting_cvv_id = content_view['versions'][0]['id']
+
+    def setUp(self):
+        """Creates export directory where the CV version contents will be exported"""
+        super(ContentViewSync, self).setUp()
+        self.export_dir = '/tmp/CvSync/{}'.format(gen_string('alphanumeric'))
```

Comment:
all you need is
```bash
$ mktemp -d -p /jyejares/custom/tmp/dir/path
/jyejares/custom/tmp/dir/path/tmp.6Q6tzvxVmh
```

Reply:
If I do this, again I have to retrive ```pwd``` for generated directory name. Better I know the name already, doesnt makes any difference in functionality though.

Reply:
what do you mean by "again"? the full path to the dir is in the "stdout" attribute of the same ssh call.
```python
In [4]: command = ssh.command('mktemp -d -p ~')
2018-11-26 13:44:47 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7fad462d9e48
2018-11-26 13:44:47 - robottelo.ssh - INFO - Connected to [sat-6-5-qa-rhel7.localhost.example.com]
2018-11-26 13:44:47 - robottelo.ssh - INFO - >>> mktemp -d -p ~
2018-11-26 13:44:48 - robottelo.ssh - INFO - <<< stdout
/home/vagrant/tmp.yKyBnALtzp

2018-11-26 13:44:48 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7fad462d9e48

In [5]: command.stdout
Out[5]: ['/home/vagrant/tmp.yKyBnALtzp', '']
```